send.ts

```diff
@@ -142,12 +141,11 @@ async function getEntity(
   let body: Uint8Array | Deno.FsFile;
   let entity: Uint8Array | FileInfo;
   const fileInfo = { mtime: new Date(mtime), size: stats.size };
-  const file = await Deno.open(path, { read: true });
   if (stats.size < maxbuffer) {
-    const buffer = await readAll(file);
-    file.close();
+    const buffer = await Deno.readFile(path);
```

Comment:
Instead of handling closing the file on `readAll`, we can just use the `readFile` api directly and remove the `readAll` dependency.